extensions/wikia/AdEngine/js/ml/rabbit.js

```diff
@@ -25,19 +25,30 @@ define('ext.wikia.adEngine.ml.rabbit', [
 		outstreamLr
 	];
 
-	function getSerializedResults() {
+	function getResults(allowedModels) {
+		allowedModels = allowedModels || [];
+
 		var results = [];
 
 		models.forEach(function (model) {
 			if (model && model.isEnabled()) {
+				if (allowedModels.length > 0 && allowedModels.indexOf(model.getName()) === -1) {
```

Comment:
Do we need to check `allowedModels.length > 0`? I believe second codnition is enough. Also you could combine it with the `if` above.

Reply:
`getSerializedResults` get predictions from all enabled models
https://github.com/Wikia/app/pull/14764/files#diff-8980c162209bf6e6a09200fc34a7b261R47